events/Random Events.txt

```diff
@@ -751,10 +736,10 @@ country_event = {
 		name = "Find a killer!"
 		random_state = {
 			rich_strata = {
-				militancy = 0.5
+				militancy = -0.5
```

Comment:
The negatives on the militancy effects are intentional -- it follows the framework that Vanilla uses for this event

Reply:
I just switched the find a killer and find the right killer militancy values, because finding some killer would imply finding some poor guy to pin it on, while finding the right killer could mean a rich person might get caught.

Reply:
This is supposed to be a retextured vanilla event. The militancy needs to follow as such